components/chart/index.tsx

```diff
@@ -26,6 +26,7 @@ function Chart({ data, token }: { config: Config; data: GraphContent[]; token?:
     message: '',
   });
   const [editMode, setEditMode] = useState(false);
+  const [searchedNode, setSearchedNode] = useState<NodeType | undefined>(undefined);
```

Comment:
```suggestion
  const [focusNode, setFocusNode] = useState<NodeType | undefined>(undefined);
```
I think this name is more logical because we are using it in the context of focusing the node here